docs/textinput-component-windows.md

```diff
@@ -0,0 +1,32 @@
+---
+id: textinput-component
+title: TextInput
+---
+
+# Reference
+
+## Props
+
+Inherits [TextInput Props](https://reactnative.dev/docs/textinput).
+
+## Windows-Specific Properties
+
+### `submitKeyEvent`
+
+A property that registers a set of KeyEvents that may trigger `OnSubmitEditing` in a multiline scenario.
+
+| type | required |
+|:--|:--|
+| { code: 'Enter', shiftKey: bool } | No |
+
+### `clearTextOnSubmit`
+
+If true, the text field will clear when submitted. The default is false.
```

Comment:
Based on phrasing of similar props:
https://reactnative.dev/docs/textinput#bluronsubmit
https://reactnative.dev/docs/textinput#cleartextonfocus-ios
```suggestion
If `true`, the text field will clear when submitted. The default value is false.
```